core/src/main/resources/error/error-classes.json

```diff
@@ -3783,6 +3783,12 @@
       "Failed to execute command because subquery expressions are not allowed in DEFAULT values."
     ]
   },
+  "_LEGACY_ERROR_TEMP_1348" : {
+    "message" : [
+      "Name <name> is ambiguous in nested CTE.",
+      "Please set <config> to CORRECTED so that name defined in inner CTE takes precedence. If set it to LEGACY, outer CTE definitions will take precedence. See more details in SPARK-28228."
```

Comment:
```suggestion
      "Please set <config> to \"CORRECTED\" so that name defined in inner CTE takes precedence. If set it to \"LEGACY\", outer CTE definitions will take precedence. See more details in SPARK-28228."
```

Reply:
> See more details in SPARK-28228.

It would be better to update the public docs and refer to them.

Reply:
I linked to the existing documentation that covers this configuration in the migration guide: https://spark.apache.org/docs/latest/sql-migration-guide.html#query-engine